pyproject.toml

```diff
@@ -20,7 +20,7 @@ boto3 = "^1.28.22"
 ops = ">=2.0.0"
 # data_platform_libs/v0/upgrade.py
 poetry-core = "*"
-pydantic = "^1.10"
+pydantic = "^1.10, <2"
```

Comment:
is the <2 needed? i thought ^1.10 will upgrade to a new version as long as the major version is `1`

Reply:
056c5f8